assets/src/patterns/components/c-button/c-button.js

```diff
@@ -1,3 +1,3 @@
 export default function cButton( props ) {
-	return <div className={ props.c_button_classes }>{ props.c_button_text }</div>;
+	return <button className={ 'c-button ' + props.c_button_classes }>{ props.c_button_text }</button>;
```

Comment:
This could _potentially_ be a generated based on the Twig, right now it is redundant